roles/nginxplus/templates/nginx.conf.j2

```diff
@@ -54,7 +54,14 @@ http {
                       '$status $body_bytes_sent "$http_referer" '
                       '"$http_user_agent" "$http_x_forwarded_for"';
 
-    access_log  /var/log/nginx/access.log  main;
+    # access_log  /var/log/nginx/access.log  main;
+    map $status $loggable {
+        ~^[23]  0;
+        default 1;
```

Comment:
It isn't obvious what this does. Can you put in a comment, or a link to the docs (in the PR if not in the code)?

Reply:
Yes it would close #3840